### ISSUE TITLE
Excessive whitespace in table cells

### ISSUE THREAD

**Issue by JayDaley** [ietf-tools/wagtail_website#76]
Table cells have a 20px padding all round, as set by:
```css
@media (min-width: 620px)
.block-table th, .block-table td {
padding: 20px;
```
This can comfortably be reduced to 10px, which is much more readable.

**Comment by JayDaley** [comment]
Still an issue following the template reimplementation

**Comment by kesara** [comment]
Released with [v1.2.0](https://github.com/ietf-tools/wagtail_website/releases/tag/v1.2.0).
